Regular Expressions/Validate File extension before Upload/readme.md

```diff
@@ -0,0 +1,33 @@
+# Regular Expression for allowing Word Document and PDF files only
+
+## Problem statement
+
+Find or detect word or pdf file before upload
+
+## Regex code explanation
+
+```js
+const regex = /([a-zA-Z0-9\s_\\.\-:])+(.doc|.docx|.pdf)$/;
```

Comment:
The use of `.` in the regex string for the file names is not escaped and thus will not behave as expected